src/components/ContractInfo.tsx

```diff
@@ -15,15 +18,47 @@ const InfoItem = ({
 );
 
 const ContractInfo = () => {
-  const { GiveStableFee, WantStableFee } =
+  const { GiveStableFee, WantStableFee, MintLimit } =
     useAtomValue(governedParamsAtom) ?? {};
-  const { anchorPoolBalance } = useAtomValue(metricsAtom) ?? {};
+  const { anchorPoolBalance, mintedPoolBalance } =
```

Comment:
should `{}` be the default value for the atom?

Reply:
I think it's okay as null